easybuild/easyconfigs/p/PyTorch/PyTorch-1.5.0-cuda-10.1.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'Conda'
+
+name = 'PyTorch'
+version = '1.5.0'
+
+homepage = 'https://pytorch.org/'
+description = """Tensors and Dynamic neural networks in Python with strong GPU acceleration.
+PyTorch is a deep learning framework that puts Python first."""
+
+toolchain = SYSTEM
+
+dependencies = [
+    ('Miniconda3', '4.7.10', '', True),
+]
+
+requirements  = ' pytorch==1.5.0=py3.7_cuda10.1.243_cudnn7.6.3_0'
+requirements += ' torchvision==0.6.0=py37_cu101'
+requirements += ' cudatoolkit==10.1.243=h6bb024c_0'
+requirements += ' future'
```

Comment:
these are the from https://github.com/pytorch/pytorch/blob/master/requirements.txt (Would you know why is conda not pulling them automatically?)

Should I fix them to the current version?

Reply:
I think it does. Maybe some of those like `future` that are not installed automatically are no longer needed?

In principle, to have reproducible recipes, the versions need to be specified. The versions can be set from the list of packages that were installed. I would say, same thing for Miniconda3.
```
    package                    |            build
    ---------------------------|-----------------
    blas-1.0                   |              mkl           6 KB
    ca-certificates-2020.1.1   |                0         125 KB
    certifi-2020.4.5.1         |           py37_0         155 KB
    cudatoolkit-10.1.243       |       h6bb024c_0       347.4 MB
    freetype-2.9.1             |       h8a8886c_1         550 KB
    intel-openmp-2020.1        |              217         780 KB
    jpeg-9b                    |       h024ee3a_2         214 KB
    ld_impl_linux-64-2.33.1    |       h53a641e_7         568 KB
    libffi-3.3                 |       he6710b0_1          50 KB
    libgfortran-ng-7.3.0       |       hdf63c60_0        1006 KB
    libpng-1.6.37              |       hbc83047_0         278 KB
    libtiff-4.1.0              |       h2733197_0         447 KB
    mkl-2020.1                 |              217       129.0 MB
    mkl-service-2.3.0          |   py37he904b0f_0         218 KB
    mkl_fft-1.0.15             |   py37ha843d7b_0         154 KB
    mkl_random-1.1.0           |   py37hd6b4f25_0         321 KB
    ncurses-6.2                |       he6710b0_1         817 KB
    ninja-1.9.0                |   py37hfd86e86_0         1.2 MB
    numpy-1.18.1               |   py37h4f9e942_0           5 KB
    numpy-base-1.18.1          |   py37hde5b4d6_1         4.2 MB
    olefile-0.46               |           py37_0          50 KB
    openssl-1.1.1g             |       h7b6447c_0         2.5 MB
    pillow-7.1.2               |   py37hb39fc2d_0         603 KB
    pip-20.0.2                 |           py37_3         1.7 MB
    python-3.7.7               |       hcff3b4d_5        45.1 MB
    pytorch-1.5.0              |py3.7_cuda10.1.243_cudnn7.6.3_0       399.5 MB  pytorch
    readline-8.0               |       h7b6447c_0         356 KB
    setuptools-46.2.0          |           py37_0         513 KB
    six-1.14.0                 |           py37_0          27 KB
    sqlite-3.31.1              |       h62c20be_1         1.1 MB
    torchvision-0.6.0          |       py37_cu101        11.8 MB  pytorch
    wheel-0.34.2               |           py37_0          51 KB
    xz-5.2.5                   |       h7b6447c_0         341 KB
    ------------------------------------------------------------
```